### ISSUE TITLE
[Accessibility]: Image without meaning in retention policy description

### ISSUE THREAD

**Issue by efritz** [sourcegraph/sourcegraph#44559]
### Audit type

Screen reader navigation

### User journey audit issue

https://github.com/sourcegraph/sourcegraph/issues/33514

### Problem description

<img width="636" alt="image" src="https://user-images.githubusercontent.com/103087/202496224-97df8e27-1f6e-4f56-83ef-c2400641a769.png">

### Expected behavior

Maybe just replace "yes (i)" with the actual alt text, it's more descriptive.

### Assigning labels

- [x] Please give this issue an estimate by applying a label like `estimate/Xd`, where X is the estimated number of days it will take to complete.
- [x] If this issue is specific to a specific Sourcegraph product, please assign the appropriate team label to this issue.
- [x] If this issue will require input from designers in order to complete, please assign the label `needs-design`.
- [x] If you are confident that this issue should be fixed by GitStart, please assign the label `gitstart`.

### Owner

This issue will be fixed by my team, I have assigned a relevant member to it, or I will do so in the near future.

**Comment by felixfbecker** [comment]
I don't see an issue with this, the element has `aria-label` so it's accessible. Closing
